contents/teams/customer-comms/objectives.mdx

```diff
@@ -1,31 +1,34 @@
-- **What we'll ship:** Zendesk improvements ([domain problem](https://github.com/PostHog/meta/issues/201), [bounced emails](https://github.com/PostHog/company-internal/issues/1324) etc.), MaxAI v2, handbook definitions (incl. definitions of what we don't support), team tracking.
-- **We'll know we're successful when:** MaxAI v2 feels worth keeping and incoming tickets aren't getting out of control.
-
-### Make sure we have successful launches for new products (Joe)
+### Improve our SLA achievement without causing a CSAT catastrophe (Ben & Ben)
+- **Rationale:** Business as usual = A good start
+- **What we'll ship:** Issue coming soon
+- **We'll know we're successful when:** We're regularly hitting 90% SLA achievement for Normal+ prio tickets. CSAT doesn't fall below 80%.
+
+### Make everyone a subject matter expert as part of their onboarding (Abigail)
+​​- **Rationale:** More experts = less escalations
+- **What we'll ship:** Issue coming soon
+- **We'll know we're successful when:** Everyone is an expert in something and we have a playbook for learning when we hire next.
+
+### Support users outside of Zendesk (Steven)
+​​- **Rationale:** Other communities = good word of mouth
+- **What we'll ship:** Issue coming soon
+- **We'll know we're successful when:** We know if a community focus is worth it and we more regularly see users helping each other. 
+
+### Centralize and improve all email comms (Joe)
+​​- **Rationale:** Better processes = Easier messaging beta
+- **What we'll ship:** Issue coming soon
+- **We'll know we're successful when:** We have all emails coming from C.io, campaign metrics generally improve, messaging beta goes well.
+
+### Make sure our upcoming betas and product launches are successful (Joe)
 - **Rationale:** More products = More revenue. 
-- **What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/215).
-- **We'll know we're successful when:** We're successfully completing all launch plans.
-
-### Continue supporting CS & Sales (Joe)
-- **Rationale:** There's a lot of things still left to do.
-- **What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/215).
-- **We'll know we're successful when:** We're effectively processing deals of all sizes and up/cross-selling users into new features.
-
-### Onboard community manager (Joe)
-- **Rationale:** More WOM = More growth
-- **What we'll ship:** [See issue](https://github.com/PostHog/meta/issues/215).
-- **We'll know we're successful when:** They've found two influencers we can work with successfully.
+- **What we'll ship:** Issue coming soon.
+- **We'll know we're successful when:** We're successfully completing all launch plans and are getting enough users in each beta. 
 
 ## Side quests
-- Run another YC S24 campaign, but with less direct work (Joe)
+- Run another YC F24 campaign, but with _even_ less direct work (Joe)
+- Continue supporting CS & Sales with everything they need to sell (Joe)
```

Comment:
This is worth doing but I don't think needs to be a specific goal unless you have particular tasks/projects in mind?

Reply:
For F24, I at least want to do the Supergroup stuff. 
For CS, mainly it'll be case studies -- Cam has been asking for non-technical ones, so I'm at least going to look at that. 

No strong feelings on having them on here or not. It'll go on my Q4 planning issue regardless.